packages/module/patternfly-docs/content/extensions/virtual-assistant/about-virtual-assistant.md

```diff
@@ -5,12 +5,23 @@ id: About chatbot
 sortValue: 1
 ---
 
-PatternFly's chatbot lives in its own package [`@patternfly/virtual-assistant`](https://www.npmjs.com/package/@patternfly/virtual-assistant)
+**Note:** The PatternFly chatbot extension lives in its own package [`@patternfly/virtual-assistant`](https://www.npmjs.com/package/@patternfly/virtual-assistant)
 
-# Virtual assistant
+It is important to leverage AI in a way that improves the experiences of your users. In alignment with our [AI principles](/patternfly-ai/about-ai#ai-principles) and [ethical design guidelines,](/patternfly-ai/ai-guidelines) we've created this chatbot extension. The PatternFly chatbot contains a React implementation of an AI chatbot, to provide your users with access to convenient and persistent help.
 
-The extension contains a React implementation of an AI chatbot.
+*Only use this chatbot when it adds value to your users' experiences.* Don't use it simply for the sake of novelty&mdash;your users will only be excited to interact with a new chatbot feature if it directly helps them reach their goals.
 
-If you notice a bug or have a suggestion for the extension, feel free to file an issue in our [GitHub repository](https://github.com/patternfly/virtual-assistant/issues)! Please make sure to check if there is already a pre-existing issue before creating a new issue.
+## How to use the chatbot
 
-To view working demos of the chatbot, view them at [Chatbot](/patternfly-ai/chatbot/chatbot/react-demos) and [Chatbot attachments](/patternfly-ai/chatbot/chatbot-attachments/react-demos).
+To illustrate the capabilities of the chatbot, we offer a [chatbot demo](/patternfly-ai/chatbot/chatbot/react-demos) and a [chatbot attachments demo](/patternfly-ai/chatbot/chatbot-attachments/react-demos). To learn how to write for chatbot experiences, refer to our [chatbot writing guidelines.](/patternfly-ai/conversation-design/#writing-for-chatbots)
```

Comment:
Chatbot demo is now at /patternfly-ai/chatbot/chatbot-container/react-demos and attachment demo is now at /patternfly-ai/chatbot/chatbot-attachments/react-demos.